src/mocks/cockpit.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+// TODO: maybe we should pull in the cockpit types here
+// and keep them in the project. Not ideal because it may
+// diverge, so we need to think about it.
+export interface UserInfo {
```

Comment:
I need to set up the tslint to enforce this but generally I'm trying to use types and not interfaces.